src/main/java/com/iridium/iridiumteams/managers/MissionManager.java

```diff
@@ -43,22 +43,25 @@ public LocalDateTime getExpirationTime(MissionType missionType, LocalDateTime st
      * Determines the missions to be checked
      *
      * @param team        The team
-     * @param environment The environment we are in
+     * @param missionWorld       The world we are in
      * @param missionType The mission type e.g. BREAK
      * @param identifier  The mission identifier e.g. COBBLESTONE
      * @param amount      The amount we are incrementing by
      */
-    public void handleMissionUpdate(T team, World.Environment environment, String missionType, String identifier, int amount) {
+    public void handleMissionUpdate(T team, World missionWorld, String missionType, String identifier, int amount) {
+
+        if (iridiumTeams.getConfiguration().whitelistedWorlds.stream().noneMatch(world -> missionWorld.getName().equalsIgnoreCase(world))) return;
```

Comment:
We probabbly also want to make it so if the list is empty, then it allows any world

Reply:
oh thats a good idea, i will do that